src/main.cpp

```diff
@@ -69,6 +70,35 @@ void discardImuMeasurements(size_t numMeasurements = 5000)
   }
 }
 
+/**
+ * Determine the initial angular velocity readings of the IMU. 
+ * This value is subtracted from the angular velocity readings of the IMU
+ * to get the correct angular velocity. 
+ * All angular velocities should be zero (at the start) when quadrotor is still.
+ */
+void initAngularVelocity()
+{
+  float AngularVelocityTemp[3] = {0};
```

Comment:
[C++ variables should be camel-case starting with a lowercase letter](https://github.com/QUB-ASL/bzzz/blob/main/CONTRIBUTING.md#31-cc)

Reply:
oops my bad I'll sort that out